composer.lock

```diff
@@ -8544,5 +8018,5 @@
         "ext-openssl": "*"
     },
     "platform-dev": [],
-    "plugin-api-version": "2.0.0"
+    "plugin-api-version": "1.1.0"
```

Comment:
It looks like composer.lock includes so many differences because it was generated using composer 1.x

Reply:
Fixed!